ibis/backends/tests/test_aggregation.py

```diff
@@ -1261,11 +1261,6 @@ def test_topk_op(alltypes, df):
         )
     ],
 )
-@mark.broken(
-    ["bigquery"],
-    raises=GoogleBadRequest,
-    reason='400 Syntax error: Expected keyword JOIN but got identifier "SEMI"',
```

Comment:
sqlglot is also giving us the `SEMI`/`ANTI` join transformation to `EXISTS`/`NOT EXISTS` as well!